Add `@recast-navigation/playcanvas` package

isaac-mason opened the pull request.

marklundin commented:
Hey @isaac-mason just checking if you need anything more from me on this. Happy to jump in to help move things forward :D

isaac-mason commented:
Hey @marklundin, I think this branch is just about ready to merge and get out there!

It would be great if you could take a look at the standalone playcanvas example I added, as well as take a look at changes I made to @recast-navigation/playcanvas. I'm not deeply familiar with playcanvas so let me know if I should revert/amend any of the changes I made.

You can see the standalone playcanvas example I added here: https://github.com/isaac-mason/recast-navigation-js/pull/438/files#diff-6eb676771678639035973890d1cfb6f747c48365ce2f343f6500725f77be662a

The main changes I made to @recast-navigation/playcanvas are:
- removing the need to pass a graphics device to the generator helpers eg `pcToSoloNavMesh`
- moving common debug draw utils into the core library
- prettier blast and small refactors
